integration-tests/schemas/createSchema.test.ts

```diff
@@ -0,0 +1,35 @@
+import "@frequency-chain/api-augment";
+import { ApiRx } from "@polkadot/api";
+import { connect } from "./scaffolding/apiConnection"
+
+import assert from "assert";
+
+import { AVRO_GRAPH_CHANGE } from "./fixtures/avroGraphChangeSchemaType";
+import { filter, firstValueFrom } from "rxjs";
+import { groupEventsByKey } from "./scaffolding/helpers";
+import { KeyringPair } from "@polkadot/keyring/types";
+
+describe("#createSchema", () => {
+    let api: ApiRx;
+    let keys: KeyringPair;
+
+    before(async () => {
+        let {api: connectApi, keys: connectKeys} = await connect(process.env.WS_PROVIDER_URL);
+        api = connectApi
+        keys = connectKeys
+    })
+
+    after(() => {
+        api.disconnect()
+    })
+
+    it("should successfully create an Avro GraphChange schema", async () => {
+        const chainEvents = await firstValueFrom(api.tx.schemas.createSchema(JSON.stringify(AVRO_GRAPH_CHANGE), "AvroBinary", "OnChain").signAndSend(keys).pipe(
+                filter(({status}) => status.isInBlock),
```

Comment:
So we need to filter on the either isInBlock or isFinalized as sometimes it can come through as isFinalized. The reasons are still a bit unclear to me, but for our testing purposes either result is fine. @rlaferla I feel like we had a reference about this that I cannot recall when working with the schemas repo... 

```suggestion
                filter(({status}) => status.isInBlock || status.isFinalized),
```

Reply:
In my tests, i noticed that inBlock and Finalized almost always happen in order (i actually have not yet witnessed a time finalized comes first). Also, Finalized is never emitted on an instant sealing node (which is where these tests will likely be run). I can include this condition, but I don't think it will be necessary.

Reply:
Without it I was stalling sometimes, but adding in this or condition fixed it for me. We can take a look at it on my box later.